### ISSUE TITLE
Error on `resume export` when `basics.summary` does not exist

### ISSUE THREAD

**Issue by christopherseaman** [rbardini/jsonresume-theme-even#10]
For making resumes without an "About" section.

To reproduce, in a clean dir:
1. run `resume init` (fill in name/email)
2. edit resume.json to delete the summary line in basics
3. run `resume export output.html -t even`
4. error occurs (output attached)
[error.txt](https://github.com/rbardini/jsonresume-theme-even/files/6105478/error.txt)

Tested and the same resume.json does export using the elegant theme and when a blank summary line (`"summary": "",`) is included although that produces an undesired blank summary line on the output as well (example).

Oddly enough, this seems to work as desired on registry.jsonresume.org (see attached pics with and without a blank summary line in the basics obj).

<img width="684" alt="with_summary" src="https://user-images.githubusercontent.com/86775/110409812-1e09f580-8045-11eb-9550-72284566fe8e.png">
<img width="686" alt="without_summary" src="https://user-images.githubusercontent.com/86775/110409814-1ea28c00-8045-11eb-978d-82ea137fdbb1.png">

**Comment by rbardini** [comment]
Thanks for the report, @christopherseaman! This has been fixed in v0.10.1.

Note that you may have to wait until the dependency version is updated in resume-cli, unless you use one of the other methods described in the README 😉

**Comment by christopherseaman** [comment]
Thanks for the (amazingly) quick response. Confirmed working.

```
resume.json on  master [✘?⇣] via ⬢ v15.11.0 on ☁️ christopher.seaman@gmail.com 
❯ resume export asdf.html -r resume_with_summary.json

Done! Find your new .html resume at:
 /Users/christopher/Documents/Development/resume.json/asdf.html

resume.json on  master [✘?⇣] via ⬢ v15.11.0 on ☁️ christopher.seaman@gmail.com 
❯ resume export asdf.html -r resume_without_summary.json

Done! Find your new .html resume at:
 /Users/christopher/Documents/Development/resume.json/asdf.html

resume.json on  master [✘?⇣] via ⬢ v15.11.0 on ☁️ christopher.seaman@gmail.com 
❯ npm list
resume.json@ /Users/christopher/Documents/Development/resume.json
└── jsonresume-theme-even@0.10.1
```
